libraries/BLE/src/BLEAdvertising.cpp

```diff
@@ -79,6 +79,41 @@ void BLEAdvertising::addServiceUUID(const char* serviceUUID) {
 } // addServiceUUID
 
 
+/**
+ * @brief Remove a service uuid to exposed list of services.
+ * @param [in] index The index of the service to stop exposing.
+ */
+bool BLEAdvertising::removeServiceUUID(int index) {
+	
+	// If index is larger than the size of the
+	// advertised services, return false
+	if(index > m_serviceUUIDs.size()) return false;
+	
+	m_serviceUUIDs.erase(m_serviceUUIDs.begin() + i);
```

Comment:
`i` should be `index`: https://github.com/espressif/arduino-esp32/actions/runs/6460395968/job/17756588546?pr=8747#step:5:173

Reply:
Thank you.  All fixed.